scripts/update-client.sh

```diff
@@ -47,6 +47,13 @@ if [[ -z ${GEN_ROOT:-} ]]; then
     GEN_ROOT="${TEMP_FOLDER}/gen"
     echo ">>> Cloning gen repo"
     git clone --recursive https://github.com/kubernetes-client/gen.git "${GEN_ROOT}"
+    # pick https://github.com/kubernetes-client/gen/pull/97
+    cd "${GEN_ROOT}"
+    git remote add tomplus https://github.com/tomplus/gen.git
```

Comment:
it is a personal repo. does openapi generator have an official repo?

Reply:
I think Roy is trying to pick the kubernetes-client/gen changes by tomplus

Reply:
Why not merge this https://github.com/kubernetes-client/gen/pull/97 it first?

Reply:
Talked with @yliaog offline. This is picking kubernetes-client/gen#97 into `gen` repo before we have it approved and merged. Timing-wise I can go with having this pick for the alpha release. For beta release we should definitely use upstream `gen` with kubernetes-client/gen#97 merged